include/robot_localization/navsat_transform.hpp

```diff
@@ -149,13 +161,13 @@ class NavSatTransform : public rclcpp::Node
    * @brief Converts the odometry data back to GPS and broadcasts it
    * @param[out] filtered_gps The NavSatFix message to prepare
    */
-  bool prepareFilteredGps(sensor_msgs::msg::NavSatFix * filtered_gps);
+  bool prepareFilteredGps(sensor_msgs::msg::NavSatFix *filtered_gps);
```

Comment:
The previously conformed to ROS 2 [standards](https://docs.ros.org/en/rolling/The-ROS2-Project/Contributing/Code-Style-Language-Versions.html#pointer-syntax-alignment), but now doesn't. Would you mind changing it back?